js/script.js

```diff
@@ -11,83 +11,83 @@
 //When user inputs text into textbox, clicking enter will submit it
 var inputByEnter = document.getElementById("textbox");
 
-inputByEnter.addEventListener("keyup", function(event){
-	//13 signifies the Enter key
-	if(event.keyCode === 13){	
-		//This cancels the default action if necessary
-		event.preventDefault();
-		//Call the addItem() function below
-		addItem();
-	}
+inputByEnter.addEventListener("keyup", function (event) {
+  //13 signifies the Enter key
+  if (event.keyCode === 13) {
+    //This cancels the default action if necessary
+    event.preventDefault();
+    //Call the addItem() function below
+    addItem();
+  }
 });
 
 
 //Create a new item on the list
-function addItem() {
+function addItem () {
```

Comment:
Refer to our discussion today on how to make this better. Overall, this is fine. Just use better names for class names and decompose this out to more functions.